alerta/app.py

```diff
@@ -44,6 +45,8 @@ def create_app(config_override=None, environment=None):
     qb.init_app(app)
     sentry.init_app(app)
 
+    auth.init_app(app)
```

Comment:
Unsure if this is the best way, however I had issues with reading the configuration any other way. 
Issue occurs as the Flask app configuration seems not to be available at import-time.